index.html

```diff
@@ -409,31 +409,60 @@ <h2>Recommended mark names</h2>
     <dl>
       <dt>"<dfn class="export">mark_fully_loaded</dfn>"</dt>
       <dd>The time when the page is considered fully loaded as marked by the
-        developer in their application.</dd>
+        developer in their application.
+        <div class="note">
+          <p>In this example, the page asynchonously initializes a chat widget, a
+            searchbox, and a newsfeed upon loading. When finished, the
+            "<a>mark_fully_loaded</a>" mark name enables lab tools and analytics
+            providers to automatically show the timing.
+          </p>
+          <pre class="example">
+                window.addEventListener("load", (event) => {
+                  Promise.all([
+                    loadChatWidget(),
+                    initializeSearchAutocomplete(),
+                    initializeNewsfeed()]).then(() => {
+                      performance.mark('mark_fully_loaded');
+                  });
+                });
+              </pre>
+        </div>
+      </dd>
       <dt>"<dfn class="export">mark_fully_visible</dfn>"</dt>
       <dd>The time when the page is considered fully visible to an end-user
         as marked by the developer in their application.</dd>
       <dt>"<dfn class="export">mark_interactive</dfn>"</dt>
       <dd>The time when the page is considered interactive to an end-user as
         marked by the developer in their application.</dd>
+      <dt>"<dfn class="export">mark_use_counter</dfn>"</dt>
```

Comment:
The terminology of `use counter` is confusing and implies that the browser will be incrementing a counter somewhere, when I don't believe that is the case for this proposal? A term that is meaningful outside of internal chromium terminology might be better - perhaps something like `mark_feature_usage`?

Reply:
That's great feedback, thanks!!

Reply:
Sorry I missed this, will update in a follow-up

Reply:
Follow-up: https://github.com/w3c/user-timing/pull/108